### ISSUE TITLE
Error when the `ON_PREM` is not set in the config

### ISSUE THREAD

**Issue by ubajze** [nautobot/nautobot-plugin-chatops-arista-cloudvision#13]
### Environment
* Python version:  3.6.13
* Nautobot version:  1.1.2
* nautobot-chatops-extension-arista version: 1.0.1

### Expected Behavior
Documentation says that the default value is `False`.

```
ON_PREM: By default this is set to False, this must be changed to True if using an on-prem instance of CloudVision.
```

### Observed Behavior
The following config is used:
```
    "nautobot_chatops_arista_cloudvision": {
        "cvaas_token": os.environ.get("ARISTA_CV_TOKEN"),
    }
```

The following error is displayed:
```
celery_1    | Traceback (most recent call last):
celery_1    |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 450, in trace_task
celery_1    |     R = retval = fun(*args, **kwargs)
celery_1    |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 731, in __protected_call__
celery_1    |     return self.run(*args, **kwargs)
celery_1    |   File "/usr/local/lib/python3.6/site-packages/nautobot_chatops/utils.py", line 40, in celery_worker_task
celery_1    |     return function(subcommand, params=params, dispatcher_class=dispatcher_class, context=context)
celery_1    |   File "/usr/local/lib/python3.6/site-packages/nautobot_chatops_arista_cloudvision/worker.py", line 73, in cloudvision_chatbot
celery_1    |     return handle_subcommands("cloudvision", subcommand, **kwargs)
celery_1    |   File "<decorator-gen-1>", line 2, in handle_subcommands
celery_1    |   File "/usr/local/lib/python3.6/site-packages/prometheus_client/context_managers.py", line 66, in wrapped
celery_1    |     return func(*args, **kwargs)
celery_1    |   File "/usr/local/lib/python3.6/site-packages/nautobot_chatops/workers/__init__.py", line 298, in handle_subcommands
celery_1    |     result = registry[command]["subcommands"][subcommand]["worker"](dispatcher, *params)
celery_1    |   File "/usr/local/lib/python3.6/site-packages/nautobot_chatops/workers/__init__.py", line 187, in wrapper
celery_1    |     return func(*args, **kwargs)
celery_1    |   File "/usr/local/lib/python3.6/site-packages/nautobot_chatops_arista_cloudvision/worker.py", line 83, in get_devices_in_container
celery_1    |     container_list = get_cloudvision_containers()
celery_1    |   File "/usr/local/lib/python3.6/site-packages/nautobot_chatops_arista_cloudvision/utils.py", line 81, in get_cloudvision_containers
celery_1    |     clnt = connect_cvp()
celery_1    |   File "/usr/local/lib/python3.6/site-packages/nautobot_chatops_arista_cloudvision/utils.py", line 63, in connect_cvp
celery_1    |     if ON_PREM.lower() == "true":
celery_1    | AttributeError: 'NoneType' object has no attribute 'lower'
```

### Steps to Reproduce
1. Run the Nautobot with the plugin. Use the provided config.
2. Run the command from Slack.
3. Check the error in the debug output.

**Comment by ubajze** [comment]
Maybe it would also be nice to make this config more robust.

I set the parameter as bool, and I get the error:

```
celery_1    |     if PLUGIN_SETTINGS.get("on_prem", "False").lower() == "true":
celery_1    | AttributeError: 'bool' object has no attribute 'lower'
```

**Comment by qduk** [comment]
Closed by PR #15
